### ISSUE TITLE
Slickdeals links blocked and put you in an endless loop

### ISSUE THREAD

**Issue by drod3763** [simonfrey/unshort.link#39]
Looks like all Slickdeals urls are blocked by the extension as a short url. Clicking the link just returns you to the unshort.link page.

Example link:
https://slickdeals.net/f/13792589-free-audible-book-for-members-pile-of-bones

![Screen Shot 2020-01-22 at 11 08 55 AM](https://user-images.githubusercontent.com/7785716/72911165-84dd2a00-3d31-11ea-8f13-9860e9e5d45e.png)

**Comment by simonfrey** [comment]
The endless redirect is weird. What browser are you using?

**Comment by drod3763** [comment]
I am on Firefox, to be fair upon further investigation it seems to be a conflict between this extension and the Temporary Containers extension. When I click the link, unshort.link opens a container, then the redirect appears to open a new tab and new container which then continues indefinitely until I turn one or the other off.

I suppose this might be mitigated by excluding unshort.link redirects from opening new tabs but I would need to research this more. It just seems odd to me that its caught by unshort.link at all given that the redirect link the extensions suggests appears to be unchanged from the original.

**Comment by simonfrey** [comment]
Unshort.link does not open the redirects in a new tab. I consider this as wontfix

**Comment by drod3763** [comment]
I guess the issue is, why does it consider this link a short link https://slickdeals.net/f/13792589-free-audible-book-for-members-pile-of-bones when its not. If I click that link, I will go to the same url. There is no redirect except for the one unshort.link makes me do. Notice the source and destination are the same, this shouldn't trigger anything imo.
